Docs/user_guide/modules/dynamicmodeler.md

```diff
@@ -0,0 +1,72 @@
+# Decimation
+
+## Overview
+
+This module provides an extensible framework for automatic processing of mesh nodes by executing "Tools" on the input to generate output mesh.
+
+Output of a tool can be used as input in another tool, which allows specification of complex editing operations. This is similar to “parametric editing” in engineering CAD software, but this module is specifically developed to work well on complex meshes used in biomedical applications (while most engineering CAD software does not directly support parametric editing of complex polygonal meshes).
+
+## Use cases
+
+* Parcellating white matter surface with curves and planes
+
+* Skull mirroring for facial deformity reconstruction
+
+## Tools
```

Comment:
@Sunderlandkyl @lassoan  I listed the additional tools added since the discourse post was created.

Did I miss any ?